django_declarative_apis/machinery/tasks.py

```diff
@@ -7,7 +7,7 @@
 
 import logging
 from pydoc import locate
-from celery.task import task as celery_task
```

Comment:
Looks `shared_task` has been around for ages. Curious as to why it wasn't originally used.